### ISSUE TITLE
Web-based playground editor

### ISSUE THREAD

**Issue by aduros** [aduros/wasm4#98]
It would be neat to have an easily accessible editor for live coding. Something like the [TypeScript Playground](https://www.typescriptlang.org/play) with the code on the left and the game on the right, with the ability to inspect the WAT and download the `cart.wasm`.

AssemblyScript seems like it would be a perfect fit for this! @MaxGraey :eyes: Maybe the existing AS editor could be repurposed?

**Comment by MaxGraey** [comment]
Yes, it possible with AS. See web sdk example: https://github.com/AssemblyScript/examples/tree/main/sdk
Also you could check how playground implemented on official page: https://github.com/AssemblyScript/website/blob/main/src/.vuepress/public/editor.html

**Comment by DenialAdams** [comment]
Supporting some languages other than AS would be awesome too - Roland's compiler can be run in the web, and I'm sure we could do WAT->WASM (this in particular was requested by Snufkin on discord)
